### ISSUE TITLE
your project in scratch is broken

### ISSUE THREAD

**Issue by scratchusernamemrtbts** [griffpatch/griffpatch.github.io#9]
[this project](https://scratch.mit.edu/projects/113321949/) is broken

**Comment by griffpatch** [comment]
Yeah I know :( - I used a hacked block to help draw the font :( - tricky to
fix!!!

**Comment by scratchusernamemrtbts** [comment]
And [this](https://scratch.mit.edu/projects/293017032) is broken too
(Some keys doesn't work)

**Comment by sfederici** [comment]
> (Some keys doesn't work)

What are the keys that don't work?

**Comment by scratchusernamemrtbts** [comment]
> > (Some keys doesn't work)
> 
> 
> 
> What are the keys that don't work?

Scratch 3 can't detect shift key and some other(s) key

**Comment by sfederici** [comment]
> Scratch 3 can't detect shift key and some other(s) key

Yes. That is why they are replaced by other keys in the remix. For example shift is replaces by #, backspace by > and canc by <. This is not ideal (as you cannot type this characters) but it works and you can replace this characters in the code with other characters that you don't usually use
